docker-entrypoint.sh

```diff
@@ -36,11 +36,13 @@ for f in /docker-entrypoint-initdb.d/*; do
 	echo
 done
 
-if [[ ! -f /db/init_done ]]; then
+initialize_db_dir() {
 	echo "No database directory. Initializing"
 	touch /db/changes.log
 	mkdir -p /db/diffs
+}
 
+setup_cookie_jar() {
```

Comment:
I do not see a lot of readability improvement by introducing those two functions and reversing the order of checks for `OVERPASS_MODE` and `-f /db/init_done`. Can you elaborate, why you think this improves readability?